bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsHash.scala

```diff
@@ -116,12 +116,12 @@ object AgsHash {
     strategyKey.foreach {
       case Pwfs1NorthKey | Pwfs1SouthKey =>
         Option(ctx.getInstrument).foreach {
-          _.pwfs1VignettingClearance.getMagnitude.## +=: buf
+          _.pwfs1VignettingClearance(ctx).getMagnitude.## +=: buf
```

Comment:
VIGNETTING

My strategy for the vignetting is, as you suggested, to pass the `ObsContext` to the `SpInstObsComp.pwfs2VignettingClearance` method. (I did the same thing for `pwfs1VignettingClearance` to be congruent, even though it isn't used). You can see that getting passed around here.